axonserver/src/test/java/io/axoniq/axonserver/message/command/LoadFactorBasedSelectorTest.java

```diff
@@ -0,0 +1,96 @@
+package io.axoniq.axonserver.message.command;
+
+import io.axoniq.axonserver.message.command.hashing.ConsistentHashRoutingSelector;
+import org.junit.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit test for {@link io.axoniq.axonserver.message.command.hashing.ConsistentHashRoutingSelector}
+ *
+ * @author Sara Pellegrini
+ */
+public class LoadFactorBasedSelectorTest {
+
+    @Test
+    public void testDistribution() {
+        ConsistentHashRoutingSelector testSubject = new ConsistentHashRoutingSelector(Integer::parseInt);
+        testSubject.register("5");
+        testSubject.register("15");
+        testSubject.register("30");
+        testSubject.register("50");
+        Map<String, AtomicInteger> counter = new HashMap<>();
+        int messages = 1_000_000;
+        for (int i = 0; i < messages; i++) {
+            String handler = testSubject.selectHandler(UUID.randomUUID().toString()).orElse(null);
+            counter.computeIfAbsent(handler, h -> new AtomicInteger()).incrementAndGet();
+        }
+        System.out.println(counter);
```

Comment:
Think the `System.out.println` can be dropped.

Reply:
yes